print/templates/pattern_only.html

```diff
@@ -0,0 +1,25 @@
+    <main class="page pattern-onlyview" id="pattern-display">
```

Comment:
simply write `<main>` without classes and id, especially not with the page class.